src/framework/audio/internal/synthesizers/fluidsynth/soundmapping.h

```diff
@@ -1000,7 +1000,10 @@ inline const ArticulationMapping& articulationSounds(const mpe::PlaybackSetupDat
     }
 
     if (setupData.category == mpe::SoundCategory::Winds) {
-        return WINDS;
+        soundId = setupData.soundId();
+        if (soundId == mpe::SoundId::Horn || (int)soundId >= (int)mpe::SoundId::Tuba) {
```

Comment:
It would be better to add std::unordered_set<SoundId> and explicitly specify there SoundId for which we want to use BRASS

Reply:
I agree, just that that's a bigger change that would require more discussion

Reply:
This change is not that big... You already specified the list, but you did it implicitly using this condition: (int)soundId >= (int)mpe::SoundId::Tuba. However, it is rather unreliable because it depends on the order of items in the SoundId enum